### ISSUE TITLE
Micro Challenge #1 : Agentic PINN pipeline

### ISSUE THREAD

**Issue by RahulSundar** [DhyutiLABS/DhyutiWalkthroughs#2]
**GOAL:**
Using Langgraph, build an agentic pipeline that automates end to end PINN execution and solving of 1D inverse problems as discussed in the WalkThrough-01. 

**References:**
https://github.com/langchain-ai/langgraph

PINNsAgent: Automated PDE Surrogation with Large Language Models 
https://arxiv.org/html/2501.12053v1

[2510.05158] Lang-PINN: From Language to Physics-Informed Neural Networks via a Multi-Agent Framework 
https://arxiv.org/abs/2510.05158

**Expected outcome:**
Given prior data, and a natural language query to solving parameter estimation, or missing data reconstruction, the agentic pipeline plans, visualises data, formulates loss, executes PINN training, validation, hyperparameter optimisation,and summarizes the findings. 

**Evaluation:**
The submissions will be judged at every step of the process - using RAGAS framework and further, the PINN workflows will be evaluated quantitatively on the basis of reconstruction /prediction errors on the test set.

*All well reviewed submissions will be hosted as a jupyternotebook / python module and the contributors spotlighted through various Dhyuti media platforms - YouTube, Linkedin, Substack and our WhatsApp channel.*

Note: 1:1 mentoring will be provided on a need basis. Feel free to reach out or drop a comment here in the issue.
